### DISCUSSION TITLE
How to open svg image directly into Excalidraw layout for editing?

### DISCUSSION THREAD

**Discussion by aravindk100** [zsviczian/obsidian-excalidraw-plugin#604] [Q&A]
I have embedded/transcluded svg files. Is there a way to open the equivalent excalidraw files by right click or other means? Right now I have to find the file relative location and then the excalidraw file nearby to open for editing.  Wondering if there is an option that I am overlooking?

**Comment by zsviczian** [comment]
No, there is no such option. If you use the Excalidraw command palette action to insert the SVG image into your document, it will add a link to the drawing in a markdown comment field under the image, so you can navigate to the original file easily.
